extensions/default/src/customizations/studyBrowserCustomization.ts

```diff
@@ -44,4 +44,32 @@ export default {
     },
   ],
   'studyBrowser.studyMode': 'all',
+  'studyBrowser.thumbnailDoubleClickCallback': {
+    callback:
+      ({ activeViewportId, servicesManager, isHangingProtocolLayout }) =>
```

Comment:
I imagine users might want to run couple of callbacks, so this should be an array